modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebarTreeView.tsx

```diff
@@ -361,7 +361,9 @@ export default function LeftSidebarTreeView({
 							>
 								<Icon symbol={TYPES_TO_SYMBOLS[type]} />
 
-								{label}
+								<span className="text-truncate" title={label}>
```

Comment:
I think here it should also have a class like `lfr-objects__model-builder-left-sidebar-item-title`, and in the end be:

`className="lfr-objects__model-builder-left-sidebar-item-title text-truncate"`

But since the class `lfr-objects__model-builder-left-sidebar-item` is only applied when the item is selected, we can discuss this better on Monday.